README.md

```diff
@@ -1,2 +1,150 @@
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-dayforce/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-dayforce)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-dayforce.svg)](https://github.com/ballerina-platform/module-ballerinax-dayforce./commits/master)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-dayforce/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-dayforce/actions/workflows/build-with-bal-test-native.yml)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+## Overview
+
+Dayforce is a comprehensive human capital management system that covers the entire employee lifecycle including HR, payroll, benefits, talent management, workforce management, and services. The entire system resides on cloud that takes the burden of managing and replicating data on-premise.
+
+The Dayforce connector allows you to access the REST API of Dayforce HCM. 
+
+This connector has been tested exclusively in the Dayforce developer sample environment. If you encounter any technical issues, please reach out to us using the links provided in the [Useful Links](#useful-links) section.
+
+## Setup guide
+
+### Step 1: Create a Dayforce account
+
+Navigate to the [Dayforce](https://developers.dayforce.com/Build/Home.aspx) website and register. Follow the instructions to create an account. If your company has already purchased a namespace in Dayforce, use that to sign up. Otherwise, you can still use their `sample` environment by selecting `sample` option.
+
+### Step 2: Obtain the user credentials to access the Dayforce API
+
+To get the credentials for the `sample` environment, follow the instructions below:
+1. Navigate to API Explorer > Employee > GET Employees.
+2. Locate the basic authentication credentials, including the `username` and `password`, for the sample environment.
+
+## Quickstart
+
+To use the `dayforce` connector in your Ballerina project, modify the `.bal` file as follows:
+
+### Step 1: Import the module
+
+Import the `ballerinax/dayforce` module into your Ballerina project.
+```ballerina
+import ballerinax/dayforce;
+```
+
+### Step 2: Instantiate a new connector
+
+Instantiate a new `dayforce:Client` giving the auth details.
+
+```ballerina
+dayforce:Client dayforce = check new (
+   {
+      auth: { 
+            username: "<username>", 
+            password: "<password>"
+      }
+   }, 
+   "https://www.dayforcehcm.com/Api");
+```
+
+### Step 3: Invoke the connector operation
+
+Now, utilize the available connector operations.
+```ballerina
+public function main() returns error? {
+    dayforce:Client dayforce = ...//
```

Comment:
it seems we have initialized 2 clients (one at the module level and one within the main function). Shall we correct this?

```suggestion
    dayforce:Client dayforce = ...//
```

Reply:
Fixed in [f54eb34](https://github.com/ballerina-platform/module-ballerinax-dayforce/pull/1/commits/f54eb34eb38de4c0777362f209ecad3871052ee2)